### ISSUE TITLE
Improve light and dark mode coloration and add background color to all tabs

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Android#51]
### Terms

- [X] I have searched [open and closed design issues](https://github.com/scribe-org/Scribe-Android/issues?q=is%3Aissue+label%3Adesign)
- [X] I agree to follow Scribe-Android's [Code of Conduct](https://github.com/scribe-org/Scribe-Android/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would finalize the conversion from light to dark mode for the Scribe-Android application. In addition, it would also be nice if Scribe Blue was made the background for the remaining tabs of the application - i.e. `Installation` and `Settings`.

- Note: This issue is blocked by #43 where the color variables will be added in a way that is sustainable for future development

### Contribution

@angrezichatterbox has expressed interest in handling this one 😊 Happy to review when it's all ready!

**Comment by andrewtavis** [comment]
Ah nice, @angrezichatterbox :) As you're on Triage rights I can assign you directly now. You should also be able to assign yourself for #43? Do you want to do one PR for that one and this one?

**Comment by angrezichatterbox** [comment]
I would make a single PR for both issues. Whichever sounds better to you. I am also okay with making separate PR if that would be better for review.

**Comment by andrewtavis** [comment]
No no, let's do a single one for both, @angrezichatterbox :) Thanks for the communication here during planning!

**Comment by angrezichatterbox** [comment]
I have made a single pull request for both issues. Please tell me if there are any changes to be made. I have tried to add all the colours from the Figma Reference. and Tried to make the dark and light mode similar to Scribe-iOS

**Comment by andrewtavis** [comment]
Thanks for this, @angrezichatterbox! Will try to review soon :)

**Comment by andrewtavis** [comment]
Closed via #53 :) Thanks so much for this, @angrezichatterbox!
